sdk/anomalydetector/ai-anomaly-detector-rest/samples-dev/createMultivariateModelSample.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import createAnomalyDetectorRestClient, {
```

Comment:
In our other Rest client samples we import the factory function without the create prefix `AnomalyDetector` and the creation would look like `const client = AnomalyDetector(...)`